.gitignore

```diff
@@ -21,3 +21,4 @@
 /.rebel_readline_history
 /package-lock.json
 /browsertest-errors
+/env/dev/resources/public/css/screen.css
```

Comment:
please put generated files under `target/`

Reply:
No this should not be under target. Generally Garden files are put under the resources-directory so that they are picked by Figwheel and friends. This follows the same practice but puts development files under the development resources instead.

Reply:
generated files should be under `target/` for e.g. cleanable builds. we have a resource directory there already `target/cljsbuild`. could easily add a `target/css` e.g.